praw/__init__.py

```diff
@@ -238,6 +238,13 @@ def config_boolean(item):
         self.https_proxy = (obj.get('https_proxy') or
                             os.getenv('https_proxy') or None)
         # We use `get(...) or None` because `get` may return an empty string
+
```

Comment:
I like the want to handle more cases. The default `praw.ini` can only be modified from within the package (people shouldn't be doing that) so I'm okay if it errors when it cannot find it.

You'll notice that PRAW makes the same assumption with `check_for_updates`.